pages/builds/docker_containerized_builds.md.erb

```diff
@@ -75,24 +67,32 @@ Once you've got everything working, you can configure your build pipeline and ru
 
 ## Configuring the build step
 
-Firstly we need to add the following environment variable to our build, either on an individual build step or as a pipeline-level environment variable:
-
-```bash
-BUILDKITE_DOCKER_COMPOSE_CONTAINER=app
-```
+To run your steps in a container, we recommend using [plugins](/docs/agent/v3/plugins). If you want to use docker without plugins, see [Using Docker without plugins](/docs/builds/docker-containerized-builds#using-docker-without-plugins)
 
-This will tell the agent to run each build step like so:
+At the start of your `pipeline.yml`, add a step to build an image with the docker plugin. In this example, we use the `docker-compose` plugin to prebuild the `app` image and upload it to a registry:
 
-```bash
-docker-compose -f docker-compose.yml run app <command>
+```
+steps:
+  - name: "Docker Build"
+    plugins:
+      docker-compose#v1.3.2:
```

Comment:
Ugh this version is going to be such a pain to keep up to date.

Reply:
Should we bother keeping it up to date? I could include a note about ensuring that you're pinning the plugin to the most recent version or something?

Reply:
Yeah, that might be good. Let's go with `v2.0.0` so it's up to date anyway!

Reply:
Added this in:
![image](https://user-images.githubusercontent.com/2074469/38970618-dc6ec98e-43d9-11e8-99b0-a3e27ae6366d.png)